mfx/optimize/pixi.toml

```
@@ -0,0 +1,21 @@
+[project]
+authors = ["MFX Team"]
+channels = ["conda-forge", "main", "r", "msys2"]
+name = "mfx-optimize"
+
```

I got the following error when trying Pixi:
```
$ pixi run python -m mfx.optimize.xopt_scans
Error: 
  × The current platform does not satisfy the minimal virtual package requirements
  ╰─▶ The project does not support 'linux-64'.
      Add it with 'pixi project platform add linux-64'.
```
```suggestion
platforms = ["win-64", "linux-64", "osx-64", "osx-arm64"]

```